libraries/Microsoft.Bot.StreamingExtensions/OperationException.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.Bot.StreamingExtensions
+{
+    internal class OperationException : Exception
+    {
+        public OperationException(string message, int statusCode, object body)
+    : base(message)
```

Comment:
This is probably the wrong ctor. You need "InnerException" in most cases for a custom exception type.